loaders/adapterLoader.js

```diff
@@ -5,9 +5,11 @@
 
 'use strict';
 
+
 const fs = require('fs');
 const blockLoader = require('block-loader');
-const adapters = require('../package.json').adapters;
+
```

Comment:
done

also added `.editorconfig` rules to the project (to help contributors submitting changes in consistent style)